school/src/main/java/com/hogwarts/school/controller/StudentController.java

```diff
@@ -19,13 +21,12 @@ public StudentController(StudentService studentService) {
 
     @PostMapping("/addStudent")
     public ResponseEntity<Student> addStudent(@RequestBody Student student) {
-        Student std = studentService.addStudent(student);
-        return ResponseEntity.ok(std);
+        return ResponseEntity.ok(studentService.addStudent(student));
     }
 
     @GetMapping("/getStudent/{studentId}")
```

Comment:
```suggestion
    @GetMapping("/{studentId}")
```
не нужно ничего писать в пути - это ошибка REST